src/Symfony/Component/DependencyInjection/Extension/ExtensionInterface.php

```diff
@@ -23,6 +23,8 @@ interface ExtensionInterface
     /**
      * Loads a specific configuration.
      *
+     * @param list<array<mixed>> $configs
```

Comment:
Is `list` actually the contract here? Looking at the implementations, we don't really care about how the passed array is keyed.

However we decide here, we should add the same type to all methods that array is being passed to, e.g.

* `Symfony\Component\DependencyInjection\Extension\Extension::processConfiguration()`
* `Symfony\Component\Config\Definition\Processor::processConfiguration()`

Reply:
Good comment. Let's avoid using `list<>` for now to not advocate over-documenting the code :)